### PR TITLE
Removing errors in Contributing.md

### LINKED ISSUE #1966: Too many Grammatical errors in Contributing.md
**Actual Behaviour**

There are too many grammatical errors in Contributing.md

**Expected Behaviour**

The file should be error-free.

**Would you like to work on the issue?**

Yes, i would like to work on the issue.

### PR DESCRIPTION
**Pull request by DhruvamSharma**
Fixed #1966 

Changes: Removed grammatical errors in the file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,15 +70,15 @@ response to any instances of unacceptable behavior.
 Project maintainers have the right and responsibility to remove, edit, or
 reject comments, commits, code, issues, and other contributions
 that are not aligned to this Code of Conduct, or to ban temporarily or
-permanently any contributor for other behaviors that they deem inappropriate,
+permanently any contributor to other behaviors that they deem inappropriate,
```
Comment (mayank8318) [inline CONTRIBUTING.md]: I think "for" is correct here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,22 +12,22 @@ Please note we have a code of conduct, please follow it in all your interactions
...
-5. *We use danger, so update go to https://travis-ci.org and login with github. Next you will see a `+` near my repositories. Click on it and select your fork enabling it for Travis. Then go the dashboard and add two environment varibales which two wont get displayed in the terminal(There's an option). The first one must be named `USERNAME` and should contain your github username. The second one should be named `API_TOKEN` and should contain a github personal token with atleast scope to public repos. It enables fast and easy moderation adequate testing.*
+5. *We use danger, so update goes to https://travis-ci.org and login with GitHub. Next, you will see a `+` near my repositories. Click on it and select your fork enabling it for Travis. Then go the dashboard and add two environment variables which two won't get displayed in the terminal(There's an option). The first one must be named `USERNAME` and should contain your GitHub username. The second one should be named `API_TOKEN` and should contain a GitHub personal token with at least scope to the public repository. It enables fast and easy moderation adequate testing.*
```
Comment (mayank8318) [inline CONTRIBUTING.md]: I think there is something wrong with this line - "add two environment variables which two won't get displayed in the terminal(There's an option)". Also, use lowercase inside the bracket.

Comment (DhruvamSharma) [reply CONTRIBUTING.md]: I think the line should have been: "add two environment variables which won't get displayed in the terminal(There's an option)"

Comment (DhruvamSharma) [reply CONTRIBUTING.md]: What should I do here @mayank8318  sir?

**Pull request closed without merging**